controllers/locker_controller.py

```diff
@@ -0,0 +1,118 @@
+import csv
+import re
+from typing import List
+from models import Locker
+from constants import LOCKER_PATH, MAX_LOCKERS
+from file_handler import read_csv, write_csv
+
+class LockerSystem:
+    def __init__(self):
+        self.lockers = self.load_lockers()
+    
+    def load_lockers(self) -> List[Locker]:
+        """사물함 데이터를 로드합니다."""
+        try:
+            # 파일이 비어있거나 헤더만 있는 경우 빈 리스트 반환
+            with open(LOCKER_PATH, newline='', encoding='utf-8') as csvfile:
+                reader = csv.DictReader(csvfile)
+                data = list(reader)
+                if not data: # 데이터 행이 없는 경우
+                    return []
+                return [Locker(id=row['id'], user_id=row['user_id']) for row in data]
+        except FileNotFoundError:
+            # 파일이 없으면 빈 리스트 반환
+            return []
+        except Exception as e:
+            print(f"[오류] 사물함 데이터를 읽는 중 오류 발생: {e}")
+            return []
+    
+    def save_lockers(self):
+        """사물함 데이터를 저장합니다."""
+        data = [{'id': l.id, 'user_id': l.user_id} for l in self.lockers]
+        try:
+            write_csv(LOCKER_PATH, data, fieldnames=['id', 'user_id'])
```

Comment:
[오류] 사물함 데이터를 저장하는 중 오류 발생: write_csv() got an unexpected keyword argument 'fieldnames'

```suggestion
            write_csv(LOCKER_PATH, data)
```